source/javascripts/refills/example.js

```diff
@@ -0,0 +1,8 @@
+$(function() {
+  $("button").click(function() {
```

Comment:
I think we should have a more scaleable way of handling selectors for bitters.
maybe data attributes would be the way to go?

like `data-componentname="role"`